### ISSUE TITLE
Add engines requirements to package.json

### ISSUE THREAD

**Issue by a-ignatov-parc** [skatejs/skatejs#291]
It will be awesome to add `engines` section to `package.json` so if user's node/npm versions does not meet requirements he will be notified.

Example:
```json
{
    "engines": {
        "node": ">=0.12.x",
        "npm": "2.x.x"
    }
}
```

P.S. Was not be able to run tests on `node@0.10.32` because of `path` module doesn't have `isAbsolute` method
```
TypeError: Object #<Object> has no method 'isAbsolute'
```

**Comment by treshugart** [comment]
+1 thanks for reporting that

**Comment by treshugart** [comment]
Released in 0.13.11.
